src/neuroconv/tools/nwb_helpers.py

```diff
@@ -47,8 +47,11 @@ def get_default_nwbfile_metadata() -> DeepDict:
 
 def make_nwbfile_from_metadata(metadata: dict) -> NWBFile:
     """Make NWBFile from available metadata."""
-    metadata = dict_deep_update(get_default_nwbfile_metadata(), metadata)
     nwbfile_kwargs = metadata["NWBFile"]
+    if "session_description" not in nwbfile_kwargs:
+        nwbfile_kwargs.update(session_description="No description.")
+    if "identifier" not in nwbfile_kwargs:
+        nwbfile_kwargs.update(identifier=uuid.uuid4())
```

Comment:
Of course, the other exception here as raised in the tests is that not every received metadata structure is guaranteed to have the `NWBFile` outer key at all - can you add safe retrieval of that?